pyanaconda/modules/payload/live/live_os.py

```diff
@@ -24,12 +24,20 @@
 
 from pyanaconda.core.signal import Signal
 from pyanaconda.core.constants import INSTALL_TREE
+from pyanaconda.core.util import getSysroot, getDirSize
+
 
 from pyanaconda.modules.common.constants.objects import LIVE_OS_HANDLER
-from pyanaconda.modules.payload.handler_base import PayloadHandlerBase
+
```

Comment:
Please, remove the empty lines between imports.

Reply:
Fixed.